tools/IceRpc.Protobuf.Tools/IceRpc.Protobuf.Tools.targets

```diff
@@ -70,6 +71,11 @@
         </ItemGroup>
     </Target>
     <Target Name="ProtoClean" BeforeTargets="Clean">
+        <OutputFileNamesTask Sources="@(ProtoFile)">
```

Comment:
I find it odd to use this new task only for clean. Why not use it for the Compile too?

Or maybe we don't need a task for this simple substitution?

Reply:
The thing is we have to convert the proto file name to PascalCase, which is not that simple to do without a C# method.

Reply:
> I find it odd to use this new task only for clean. Why not use it for the Compile too?

To avoid running two tasks, as we are already have to run the compile task